manifests/cf-manifest/runtime-config/datadog-addon.yml

```diff
@@ -0,0 +1,6 @@
+addons:
+  - name: datadog-agent
```

Comment:
I think we should add `- (( append ))`  before this, to be sure that he addons are merged properly. https://github.com/geofffranks/spruce#what-about-arrays